### PR TITLE
feat: Added quake3-to-source-guide and images

### PR DESCRIPTION
**Pull request by TheNerdMan**
Mac couldn't get the local instance to work so I'm here doing this PR and MD files for him. 

Side note: Its so needlessly compliclated to create a doc page for a non developer wtf

### CODE REVIEW

**Comment by hexaflexahexagon** [comment]
You can just use the github web gui for basic pages with no images, I've done a few that way.

**Comment by braem** [comment]
What's the hard part? Should just need to install ruby & jekyll, clone, bundle, then run jekyll serve.

**Comment by TheNerdMan** [comment]
I think my comment about the complexity of it is coming from a non dev view point. To install all those things to just get a page up and running. Maybe I'm just being pedantic but @hexaflexahexagon's comment about using github's web GUI is probably a good thing that could maybe be added to the [How to Create a Documentation Page](https://docs.momentum-mod.org/guide/create-docs-page/) for those who need to make edits but don't want to have to go through the local setup

File: _posts/guides/2022-04-10-guide_porting_quake3-to-source.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+permalink: /guide/porting-quake3-maps-to-source/
+
+category: guide
+
+tags:
+  - quake3
+  - mapping
+  - maps
+  - source  
+  - porting
```
Comment (braem) [inline _posts/guides/2022-04-10-guide_porting_quake3-to-source.md:13]: Some of these tags are redundant
```suggestion
  - mapping
  - porting
```

**Review by braem — CHANGES_REQUESTED** [review]
Well written! Just trying to keep the number of tags low so our https://docs.momentum-mod.org/tags/ page doesnt become a mess.

Also could definitely be utilizing notices more, eg:

```
{:.notice--info}
blah blah blah
```
but looks like this guide will be touched again anyhow for when haste and other things are added.

**Comment by TheNerdMan** [comment]
> Also could definitely be utilizing notices more
Okie dokie, this is Mac's article so I'll go over it with him to figure out where it would make sense to add the notices. Do you have any suggestions that I can bring up with him?

**Comment by braem** [comment]
> > Also could definitely be utilizing notices more
> 
> Okie dokie, this is Mac's article so I'll go over it with him to figure out where it would make sense to add the notices. Do you have any suggestions that I can bring up with him?

Fine for now, just in the future it's good to know that we have them (info in blue, warning in yellow, and danger in red) in the case we want to highlight something in particular. They're specific to jekyll's markdown iirc (kramdown?) so it's hard to preview it without a working setup though.

**Review by braem — APPROVED** [review]

**Pull request merged**